Car.cpp

```diff
@@ -0,0 +1,16 @@
+#include "Car.hpp"
+#include <iostream>
+
+Car::~Car()       { std::cout << __FUNCTION__ << std::endl; }
+void Car::turnLeft()      { std::cout << __FUNCTION__ << std::endl; }
+void Car::turnRight()     { std::cout << __FUNCTION__ << std::endl; }
+void Car::brake() {
+    std::cout << __FUNCTION__ << std::endl;
+    speed_ = 0;
+}
+void Car::accelerate(int speed) {
+    if(speed > 0) {
+        std::cout << __FUNCTION__ << std::endl;
+        speed_ = speed;
+    }
```

Comment:
What do you think about making an exception here? Ofc when speed <=0

Reply:
Why not